core/api.js

```diff
@@ -8,8 +8,8 @@ import {UserFlow, auditGatherSteps} from './user-flow.js';
 import {snapshotGather} from './gather/snapshot-runner.js';
```

Comment:
How do we feel about keeping `core/fraggle-rock/api.js` as an alias for `core/api.js` in 10.0?

Reply:
nah doesn't seem necessary